To-do.md

```diff
@@ -0,0 +1,3 @@
+# To Do
+
+- No outstanding issues
```

Comment:
I don't think this file should be here at all — we use GitHub Issues rather than storing them in the repo. (There are lots of outstanding issues! :) )

Reply:
I deleted it.

Reply:
I still see it! Sorry to be a stickler for this